mavlink/modules/flight_controller.py

```diff
@@ -235,3 +235,12 @@ def set_flight_mode(self, mode: str) -> bool:
             print("ERROR: an unsupported flight mode is set by dronekit.VehicleMode()")
             return False
         return True
+
+    def get_flight_mode(self) -> "tuple[bool, str | None]":
```

Comment:
Return an enumeration instead of string.

Reply:
see changes